catalog/ftp/pom.xml

```diff
@@ -132,7 +132,7 @@
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.57</minimum>
+                                            <minimum>0.55</minimum>
```

Comment:
Any chance we can increase this? If not and there's a good reason, can we add a comment explaining why?